### ISSUE TITLE
Evaluate applicability of CII-Best-Practices check on GitLab repos

### ISSUE THREAD

**Issue by raghavkaul** [ossf/scorecard#3098]
This is a tracking issue to ensure this check works on GitLab repos.

To validate, the following must be complete:
- [x] Check must run successfully (without errors) in nightly cron
- [ ] Scores should be validated to make sure they generally align with the security practices of the repo (i.e. , should not be 0 when practices are clearly followed, or 10 if practices are clearly not). 
- [x] Major bugs/scoring issues should be fixed
- [x] Minor parity differences between GitHub and GitLab clients should be documented in checks.yaml and tracked as separate issues

**Comment by raghavkaul** [comment]
In cron results, we haven't found any GitLab repos that get a high score the CII-Best-Practices check. It's unclear whether this option has been publicized/is relevant for GitLab repos.

**Comment by david-a-wheeler** [comment]
The best practices badge is specifically designed so it can work with any host, including gitlab. So it definitely applies. We've focused more on projects hosted on GitHub, because there are more of them, but that's not a matter of applicability.

**Comment by david-a-wheeler** [comment]
It still applies. The best practices badge can work with basically any hosting system. The BP badge *automates* much more on GitHub. There are more projects on GitHub than GitLab, so we focused there. Neverthess, it still applies.

We've been taking steps to make more people aware of it. We did a talk in Bilbao (*not* in the OpenSSF track), and a new OpenSSF blog post is coming out.

I suggest closing this issue. We can *always* do more awareness efforts, but I don't think this suggests anything different be done.
